apps/native/app/src/screens/vaccinations/vaccinations.tsx

```diff
@@ -0,0 +1,137 @@
+import { GeneralCardSkeleton, Heading, TabButtons, Typography } from '@ui'
+import React, { useCallback, useState } from 'react'
+import { FormattedMessage, useIntl } from 'react-intl'
+import { RefreshControl, SafeAreaView, ScrollView, View } from 'react-native'
+import { NavigationFunctionComponent } from 'react-native-navigation'
+import styled from 'styled-components/native'
+
+import { useGetVaccinationsQuery } from '../../graphql/types/schema'
+import { createNavigationOptionHooks } from '../../hooks/create-navigation-option-hooks'
+import { useConnectivityIndicator } from '../../hooks/use-connectivity-indicator'
+import { VaccinationsCard } from './components/vaccination-card'
+
+const Host = styled(SafeAreaView)`
+  padding-horizontal: ${({ theme }) => theme.spacing[2]}px;
+  margin-bottom: ${({ theme }) => theme.spacing[4]}px;
+`
+
+const Vaccinations = styled.View`
+  margin-top: ${({ theme }) => theme.spacing[3]}px;
+`
+
+const Tabs = styled.View`
+  margin-top: ${({ theme }) => theme.spacing[3]}px;
+`
+
+const { getNavigationOptions, useNavigationOptions } =
+  createNavigationOptionHooks((theme, intl) => ({
+    topBar: {
+      title: {
+        text: intl.formatMessage({ id: 'health.vaccinations.screenTitle' }),
+      },
+    },
+    bottomTabs: {
+      visible: false,
+      drawBehind: true,
+    },
+  }))
+
+export const VaccinationsScreen: NavigationFunctionComponent = ({
+  componentId,
+}) => {
+  useNavigationOptions(componentId)
+  const intl = useIntl()
+  const [refetching, setRefetching] = useState(false)
+  const [selectedTab, setSelectedTab] = useState(0)
+
+  const vaccinationsRes = useGetVaccinationsQuery()
+
+  const vaccinations = vaccinationsRes.data?.healthDirectorateVaccinations
```

Comment:
nit: store this in variable for easer read and usage `vaccinationsRes.data?.healthDirectorateVaccinations
    .vaccinations`